packages/esm-appointments-app/src/types/index.ts

```diff
@@ -199,3 +199,11 @@ export interface Identifier {
   identifier: string;
   identifierName?: string;
 }
+
+export interface DailyAppointmentsCountByService {
+  appointmentDate: string;
+  services: Array<{
+    serviceName: string;
+    count: number;
+  }>;
+}
```

Comment:
this basically just types the existing data structure, so though I did change "service" to "services" to make it more clear.